galata/test/documentation/general.test.ts

```diff
@@ -487,6 +487,18 @@ test.describe('General', () => {
     ).toMatchSnapshot('command_palette.png');
   });
 
+  test('Keyboard Shortcuts Help', async ({ page, tmpPath }) => {
+    await page.goto(`tree/${tmpPath}`);
+
+    await page.notebook.createNew();
+
+    await page.keyboard.press('Control+Shift+H');
+
+    await expect(page.locator('.jp-Dialog')).toHaveScreenshot(
```

Comment:
This is a bit to large and will bring trouble, let's reduce the captured area:

```suggestion
    expect(await page.locator('.jp-Notebook').screenshot()).toMatchSnapshot(
```